corehq/apps/data_interfaces/models.py

```diff
@@ -84,6 +85,7 @@ def _try_date_conversion(date_or_string):
     return date_or_string
 
 
+@audit_fields("active", "deleted")
```

Comment:
Not sure if you've looked into the `audit_special_queryset_writes` keyword argument here, but we generally always want it in order to ensure audit events can never be accidentally missed if someone uses any of `.bulk_create()`, `.bulk_update()`, `.delete()` or `.update()` QuerySet methods.  Adding it comes with the extra work of ensuring that the `audit_action` keyword argument gets specified anywhere these methods are currently used in HQ (for this model).

```suggestion
@audit_fields("active", "deleted", audit_special_queryset_writes=True)
```

Reply:
See cdbb3cbd8d87eb1cd9c81515fdbc69bbf42cb91c (though I did something bad to tests 😅)